### PR TITLE
Quickly set QR size

### PR DESCRIPTION
**Pull request by alitelli**

### CODE REVIEW

File: QRCode.class.php [hunk 1]
```diff
@@ -340,9 +340,9 @@ public function get($iSize = 150, $sECLevel = 'L', $iMargin = 1)
      *
```
Comment (pH-7) [inline QRCode.class.php:340]: Can you please update the docBlock with `@param int $iSize`?

**Review by pH-7 — CHANGES_REQUESTED** [review]
Thanks very much for your contribution! 🎉 Really appreciated 💯 Before I can approve it, please see my comment below. Thanks!

File: QRCode.class.php (hunk 1 above)
Comment (alitelli) [reply QRCode.class.php:340]: Now better

**Comment by pH-7** [comment]
Thanks for this @alitelli 😄 Highly appreciated it!

**Pull request merged**